app/src/main/java/com/pajato/argus/Database.kt

```diff
@@ -0,0 +1,89 @@
+package com.pajato.argus
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import android.provider.BaseColumns
+
+/* Object that defines the table contents */
+object DatabaseEntry : BaseColumns {
+    val TABLE_NAME = "video"
+    val COLUMN_NAME_TITLE = "title"
+    val COLUMN_NAME_NETWORK = "network"
+    @Suppress("ObjectPropertyName")
+    val _ID = BaseColumns._ID
+}
+
+/* Helper object with our shorthand database methods */
+object DatabaseHelper {
+    fun writeVideo(v: Video, context: Context) {
+        // Add the new entry into the database
+        val db = DatabaseReaderHelper(context).writableDatabase
+        val values = ContentValues()
+        values.put(DatabaseEntry.COLUMN_NAME_TITLE, v.title)
+        values.put(DatabaseEntry.COLUMN_NAME_NETWORK, v.network)
+        /*val newRowId: Long = */db.insert(DatabaseEntry.TABLE_NAME, null, values)
+    }
+
+    fun getVideosFromDb(context: Context): MutableList<Video> {
+        // Search the database for all entries.
+        val db = DatabaseReaderHelper(context).readableDatabase
+        val projection = arrayOf(DatabaseEntry._ID, DatabaseEntry.COLUMN_NAME_TITLE, DatabaseEntry.COLUMN_NAME_NETWORK)
+        val sortOrder = DatabaseEntry.COLUMN_NAME_NETWORK + " DESC"
+        val cursor: Cursor = db.query(DatabaseEntry.TABLE_NAME, projection, null, null, null, null, sortOrder)
+        val items = mutableListOf<Video>()
+        // Put each of the database entries into Video objects and our video list.
+        while (cursor.moveToNext()) {
+            //val itemId: Long = cursor.getLong(cursor.getColumnIndexOrThrow(DatabaseEntry._ID))
+            val title = cursor.getString(cursor.getColumnIndexOrThrow(DatabaseEntry.COLUMN_NAME_TITLE))
+            val network = cursor.getString(cursor.getColumnIndexOrThrow(DatabaseEntry.COLUMN_NAME_NETWORK))
+            val video = Video(title, network)
+            items.add(video)
+        }
+        cursor.close()
+        return items
+    }
+
+    fun deleteVideo(v: Video, context: Context) {
+        // Delete a specific video entry from the database.
+        val db = DatabaseReaderHelper(context).writableDatabase
+        val selection: String = DatabaseEntry.COLUMN_NAME_TITLE + " LIKE ?"
+        val args: Array<String> = arrayOf(v.title)
+        db.delete(DatabaseEntry.TABLE_NAME, selection, args)
+    }
+
+    fun deleteAll(context: Context) {
+        val db = DatabaseReaderHelper(context).writableDatabase
+        db.delete(DatabaseEntry.TABLE_NAME, null, null)
+    }
+}
+
+/* Required SQLite implementation. */
```

Comment:
Please make top level comments be javadoc comments: /** ... */ so that they appear in generated documentation.

Reply:
Will do.